Update driver for 0.8 client pravega

dadlex opened the pull request.

dadlex committed:
Fix links for 0.8 client pravega

dadlex committed:
more link fixws

veronikaKochugova commented:
👍

dadlex committed:
more link fixes

dadlex committed:
turn off byteStream support

dadlex committed:
Use no-netty api controller

Update grpc library

AndreyKoltsov1997 commented on gradle.properties:
```diff
@@ -1,2 +1,2 @@
-pravegaCommit=4bf247f
-pravegaVersion=0.7.0-rc0
+pravegaCommit=355579c
+pravegaVersion=0.8.0-pre-release
```
I think we should change that configuration with respect to Pravega 0.8.0 release information: https://github.com/pravega/pravega/releases/tag/v0.8.0
```
pravegaCommit=e4c436b
pravegaVersion=v0.8.0
```

dadlex replied:
fair enough

AndreyKoltsov1997 commented on src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java:
```diff
@@ -364,6 +363,7 @@ public PravegaStorageDriver(
 			this.transactionMode = eventConfig.boolVal("transaction");
 		} else {
 			this.transactionMode = false;
+			throw new IllegalConfigurationException("ByteStreams are not currently supported. Contact the dev team.");
```
Please, add this information into README.md's `Byte Stream` section as well: https://github.com/emc-mongoose/mongoose-storage-driver-pravega/tree/PRAVEGA-101-update-for-0.8-pravega#52-byte-stream-operations

dadlex committed:
Use the official 0.8.0 commit

Update readme with info about unsupported byte streams
Comment out gitlab ci tags

AndreyKoltsov1997 reviewed (approved):
LGTM assuming Pravega's tags have been commented out intentionally.

dadlex committed:
Update test suite in ci for 0.8

dadlex committed:
Update version

dadlex committed:
Update pravega version in tests

Add instruction for gitlab runner

dadlex committed:
Update pravega read test

Update read mode paragraph in readme

dadlex committed:
Fix pravega read test

dadlex committed:
Fix pravega read test
